### PR TITLE
Store command statuses

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**

### CODE REVIEW

**Comment by alexander-yevsyukov** [comment]
@AlexanderLitus, please see TODOs related to storages. If you can address them in this PR, it would be good.

File: client/src/main/proto/spine/validation.proto [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    // The default format string for missing field. The parameter is the name of the missing field.
+    option (default_message) = "Field %s must be set";
+
+    bool value = 1;
+
+    // User-defined validation error message format.
+    string msg = 2;
+}
+
+message DecimalOption {
```
Comment (AlexanderLitus) [inline client/src/main/proto/spine/validation.proto]: Why `DecimalOption` instead of `NumberRange` or something?

Comment (alexander-yevsyukov) [inline client/src/main/proto/spine/validation.proto]: We try follow naming of Java Validation. See https://github.com/SpineEventEngine/core-java/wiki/Proposal-for-validation-attributes.

**Comment by AlexanderLitus** [comment]
@alexander-yevsyukov, PTAL. If it is OK, then this PR can be merged.

**Comment by alexander-yevsyukov** [comment]
@AlexanderLitus, please have `checkTimestamp()` in `Validate` class.

**Pull request merged**